app/src/main/java/org/literacyapp/authentication/StudentImageCollectionActivity.java

```diff
@@ -61,6 +62,7 @@
     private static final boolean DIAGNOSE_MODE = true;
     private static final long TIMER_DIFF = 200;
     private static final int NUMBER_OF_IMAGES = 20;
+    private static final int MAX_TIME_BEFORE_FALLBACK = 5000;
```

Comment:
Does this mean that the Student has 5 seconds to successfully place the face within the frame?